File: TypeScript/start-code/startcode.ts
// Beschreibung: Ein einfaches TypeScript-Programm
// Datum: 25.01.2025

// Begrüßung
console.log("Willkommen zu meinem TypeScript-Programm!");

// Funktion definieren
function begruessung(): void {
    console.log("Das ist eine Beispiel-Funktion in TypeScript.");
}

// Funktion aufrufen
begruessung();

// Nutzereingabe (für Node.js)
import * as readlineSync from 'readline-sync';

const name: string = readlineSync.question("Bitte geben Sie Ihren Namen ein: ");

// Begrüßung mit dem Namen
console.log(`Hallo, ${name}! Schön, dass Sie hier sind.`);

// Programm beenden
console.log("Das Programm wurde beendet.");

File: R/start-code/startcode.R
# Beschreibung: Ein einfaches R-Programm
# Datum: 25.01.2025

# Begrüßung
cat("Willkommen zu meinem R-Programm!\n")

# Funktion definieren
begruessung <- function() {
  cat("Das ist eine Beispiel-Funktion in R.\n")
}

# Funktion aufrufen
begruessung()

# Nutzereingabe
name <- readline(prompt = "Bitte geben Sie Ihren Namen ein: ")

# Begrüßung mit dem Namen
cat("Hallo, ", name, "! Schön, dass Sie hier sind.\n", sep="")

# Programm beenden
cat("Das Programm wurde beendet.\n")
